api/v2/checluster_types.go

```diff
@@ -479,6 +479,12 @@ type PersistentHomeConfig struct {
 	// Must be used with the 'per-user' or 'per-workspace' PVC strategy in order to take effect.
 	// Disabled by default.
 	Enabled *bool `json:"enabled,omitempty"`
+	// Determines whether the init container that initializes the persistent home directory should be disabled.
+	// When the `/home/user` directory is persisted, the init container is used to initialize the directory before
+	// the workspace starts. If set to true, the init container will not be created.
+	// This field is not used if the `workspace.persistUserHome.enabled` field is set to false.
```

Comment:
This sentence needs to be adapted for the Che Cluster CR: `workspace.persistUserHome.enabled` -> `devEnvironments.persistUserHome.enabled`

Reply:
Fixed